index.html

```diff
@@ -0,0 +1,219 @@
+<!DOCTYPE html>
+<html lang="en">
+
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="x-ua-compatible" content="ie=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+
+		<title>Anna Russett</title>
+	
+		<link rel="icon" type="image/x-icon" href="http://www. /favicon.ico">
+		<meta name="description" content="content">
```

Comment:
let's put the line from the homepage + the first paragraph of the bio as the content for this description meta tag, something like this:

> I help people et the most out of social media and the Internet. Let's collaborate! At school they called me a new media artist, on YouTube they call me a creator, on Instagram they call me an influencer, and in the advertising world they call me a strategist. I’ve gained an audience of over 200,000 across online platforms where I create video, photography, text and code to share vlogs, tutorials, internet and social media art that encourages other young women to be critical of and take part in shaping the world around us.